content/en/tracing/setup_overview/setup/dotnet-framework.md

```diff
@@ -219,57 +230,63 @@ To configure the Tracer using a JSON file, create `datadog.json` in the instrume
 
 {{< /tabs >}}
 
-### Configuration Variables
+## Configuration settings
 
-The following tables list the supported configuration variables. Use the first name (e.g. `DD_TRACE_AGENT_URL`) when setting environment variables or configuration files. The second name, if present (e.g. `AgentUri`), indicates the name the `TracerSettings` propery to use when changing settings in the code.
+Using the methods described above, customize your tracing configuration with these settings. Use the first name (for example, `DD_<INTEGRATION>_ENABLED`) when setting environment variables or configuration files. The second name (for example, `Enabled`), indicates the name for the `IntegrationSettings` property to use when changing settings in the code. Access these properties through the `TracerSettings.Integrations[]` indexer. 
 
-#### Unified Service Tagging
+### Unified Service Tagging
 
-| Setting Name                                        | Description                                                                                                                                                                                                       |
+To use [Unified Service Tagging][7], configure the `DD_ENV`, `DD_SERVICE`, and `DD_VERSION` settings for your services. 
```

Comment:
We shouldn't refer to the `DD_` settings here, since the name is different when configuring via code, e.g. `DD_ENV` vs `Environment`.

```suggestion
To use [Unified Service Tagging][7], configure the following settings for your services. 
```